albumentations/augmentations/transforms.py

```diff
@@ -2412,6 +2414,7 @@ def __init__(
         cutout_threshold: ScaleFloatType = 0.68,
         intensity: ScaleFloatType = 0.6,
         mode: Union[str, Sequence[str]] = "rain",
+        color: Optional[Sequence[int]] = None,
```

Comment:
I think if the `mode` is `Sequence`  we need to use `color: Dict[str, Sequence[int]]` where key is mode name. It is needed because we can not use the same color for rain and mud.

Reply:
Makes sense!

Reply:
Added support for Dict.